About-Page/about.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <script
+      src="https://kit.fontawesome.com/eca3ff88d7.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700&family=Lato:ital,wght@0,400;0,700;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="../css/shared.css" />
+    <link rel="stylesheet" href="../css/about.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <nav class="header-nav">
```

Comment:
Order for our design to match the Behance design, for the top nav,

- [ ] Please reduce its height;
- [ ] Kindly change its colour to a darker shade;

Your design:
![image](https://user-images.githubusercontent.com/83596135/157765338-4ed29989-6a80-4e59-9013-5bd40cd996c2.png)
Behance design:
![image](https://user-images.githubusercontent.com/83596135/157765455-89b17c2d-d03f-432e-803a-18c77483bafc.png)